docs/pages/component/events.md

```diff
@@ -493,7 +493,24 @@ Example:
 }
 ```
 
-Platforms: Android
+### `onSubtitleTracks`
```

Comment:
This Pr looks good to me, I am just not sure about this naming,
onTextTracks event is send when track list is updated, here I would expect the callback is thrown when subtitle track list is updated, but this is the new functionality here.
what about naming it "onTextTrackDataChanged" ?

Reply:
This makes perfect sense, thank you, I'm about to push these changes. I think it came from a misunderstanding of what `onTextTracks` was; so, thank you for clearing that up 🙌🏽